### PR TITLE
Adding new exercises and add some notes for network2

### PR DESCRIPTION
**Pull request by TGLuis**
During the session, the students took back the notes from github and put it on overleaf. I apparently was the only one to contribute. Here is the modifications I made in the notes. 
I also added a file to have exercises and corrections of the first chapter. Unfortunately nobody wanted to write the corrections of the exercises I created (the first one is corrected by me) but I think it still can be useful to put these exercises on github for the following years.

### CODE REVIEW

File: src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
 
-\hypertitle{Computer networks: configuration and management}{8}{INGI}{2142}
-{Houtain Nicolas, Gorby Nicolase Kabasele Ndondas}
+\usepackage{ragged2e}
+
+\newcolumntype{L}[1]{>{\justify\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
+\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
+\newcolumntype{R}[1]{>{\justify\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
+
+\hypertitle{network2}{7}{INGI}{2142}
```
Comment (MartinBraquet) [inline src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex]: Ici, c'est mieux de mettre le vrai nom (pas `network2`).

**Comment by TGLuis** [comment]
Pourquoi cette branche n'a toujours pas été merge? @MartinBraquet ?

**Comment by MartinBraquet** [comment]
Parce qu'il n'y a que moi qui ai approuvé :) .
________________________________

File: src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex [hunk 2]
```diff
@@ -1,22 +1,36 @@
...
+\usepackage{ragged2e}
+
+\newcolumntype{L}[1]{>{\justify\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
+\newcolumntype{C}[1]{>{\centering\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
+\newcolumntype{R}[1]{>{\justify\let\newline\\\arraybackslash\hspace{0pt}}m{#1}}
+
+\hypertitle{Computer networks: configuration and management}{7}{INGI}{2142}
+{Houtain Nicolas, Gorby Nicolase Kabasele Ndondas, Luis Tascon Gutierrez}
 {Olivier Bonaventure}
 $$$$
```
Comment (Jimvy) [inline src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex]: Mais qu'est-ce que c'était que ceci ?

File: src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex [hunk 3]
```diff
@@ -203,90 +166,67 @@ \subsection{IP precedence and ToS}
...
+		\end{itemize}
+		
+	\end{itemize}
+	
+	\item Receiver replies back with \textbf{RESV message} that carries :
+	\begin{itemize}
+		\item Rspec : Reservation specification
+	\end{itemize}
+	
+	\item[Accepted] : if accepted by all routers along the path. $\to$ Setut actual reservation and filters ont these router
```
Comment (Jimvy) [inline src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex]: J'ai quelques problèmes avec cette phrase...

File: src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex [hunk 4]
```diff
@@ -44,68 +58,41 @@ \subsubsection{Real-time application and QoS need}
...
-            (Need bandwidth to be efficient and minimun bandwidth to
-            work)
-    \end{itemize}
-
-    \item \begin{itemize}
-        \item Multimedia : 
-        \item Computation :
-    \end{itemize}
-\end{enumerate}
+\begin{tabular}{R{0.48\linewidth} C{0.04\linewidth} L{0.48\linewidth}}
```
Comment (Jimvy) [inline src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex]: Il y a un ou deux soucis de overfull hbox

File: src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex (hunk 2 above)
Comment (TGLuis) [reply src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex]: Ceci quoi?

Comment (MartinBraquet) [reply src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex]: 4 $ qui n'ont surement aucun intérêt ...

File: src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex (hunk 3 above)
Comment (TGLuis) [reply src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex]: En mettant "Setup" et "on", cela t'irait mieux?

Comment (Jimvy) [reply src/q7/network2-INGI2142/notes/network2-INGI2142-notes.tex]: Oui ;)

**Pull request merged**